### PR TITLE
Style optimizations

### PR DESCRIPTION
**Pull request by KobiHay**

### CODE REVIEW

File: our-brothers-web/src/app/auth/login-popup/login-popup.component.scss [hunk 1]
```diff
@@ -5,19 +5,28 @@
   transform: translateY(-50%);
   max-width: 500px;
 
+  @media only screen and (max-width: 768px) {
+    width: 80%;
```
Comment (gilhanan) [inline our-brothers-web/src/app/auth/login-popup/login-popup.component.scss:9]: use `max-width`

File: our-brothers-web/src/app/shared/components/registration-progress-bar/registration-progress-bar.component.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<div class="progressbar-container">
+<div class="progressbar-container container">
```
Comment (gilhanan) [inline our-brothers-web/src/app/shared/components/registration-progress-bar/registration-progress-bar.component.html:1]: `container progressbar-container`

Comment (gilhanan) [reply our-brothers-web/src/app/shared/components/registration-progress-bar/registration-progress-bar.component.html:1]: Usually, the common class comes before the specific one

File: our-brothers-web/src/app/shared/components/meetings/meetings-list/meetings-list.component.scss [hunk 3]
```diff
@@ -14,6 +18,15 @@
   .join-col {
     flex: 0 1 130px;
   }
+
+  .col {
```
Comment (gilhanan) [inline our-brothers-web/src/app/shared/components/meetings/meetings-list/meetings-list.component.scss:22]: Did you test it with populated meetings?

File: our-brothers-web/src/app/shared/components/checkbox/checkbox.component.scss [hunk 4]
```diff
@@ -17,7 +17,8 @@
 
   .site-checkbox-label {
     font-family: 'Heebo';
-    color: rgb(17, 17, 17);
+    color: #818181;
+    width: 100%;
```
Comment (gilhanan) [inline our-brothers-web/src/app/shared/components/checkbox/checkbox.component.scss:21]: Is it necessary?

File: our-brothers-web/src/app/shared/components/free-text-filter/free-text-filter.component.scss [hunk 5]
```diff
@@ -5,12 +5,20 @@
   background-color: rgb(0, 160, 153);
   box-shadow: 0px 1px 10px 0px rgba(1, 1, 1, 0.15);
 
+  @media only screen and (max-width: 768px) {
+    border: 7px solid rgb(0, 160, 153);
```
Comment (gilhanan) [inline our-brothers-web/src/app/shared/components/free-text-filter/free-text-filter.component.scss:9]: `border-width: 7px;`

File: our-brothers-web/src/app/shared/components/view-toggle/view-toggle.component.html [hunk 6]
```diff
@@ -1,14 +1,30 @@
 <div class="view-toggle-container">
```
Comment (gilhanan) [inline our-brothers-web/src/app/shared/components/view-toggle/view-toggle.component.html:1]: There is a duplication of code here. Please add a TODO comment with the message: 
``` html
<!-- TODO: Angular templates optimizations -->
```

File: our-brothers-web/src/app/shell/header/header.component.scss [hunk 7]
```diff
@@ -12,7 +12,7 @@ $mobileBreakpoint: 1200px;
 
     %bar {
       width: 30px;
-      height: $barHeight;
```
Comment (gilhanan) [inline our-brothers-web/src/app/shell/header/header.component.scss:15]: Can we update `$barHeight` value to `3px`?

File: our-brothers-web/src/app/auth/login-popup/login-popup.component.scss (hunk 1 above)
Comment (KobiHay) [reply our-brothers-web/src/app/auth/login-popup/login-popup.component.scss:9]: fix

File: our-brothers-web/src/app/shared/components/registration-progress-bar/registration-progress-bar.component.html (hunk 2 above)
Comment (KobiHay) [reply our-brothers-web/src/app/shared/components/registration-progress-bar/registration-progress-bar.component.html:1]: fix

File: our-brothers-web/src/app/shared/components/meetings/meetings-list/meetings-list.component.scss (hunk 3 above)
Comment (KobiHay) [reply our-brothers-web/src/app/shared/components/meetings/meetings-list/meetings-list.component.scss:22]: no

File: our-brothers-web/src/app/shared/components/checkbox/checkbox.component.scss (hunk 4 above)
Comment (KobiHay) [reply our-brothers-web/src/app/shared/components/checkbox/checkbox.component.scss:21]: yes

File: our-brothers-web/src/app/shared/components/free-text-filter/free-text-filter.component.scss (hunk 5 above)
Comment (KobiHay) [reply our-brothers-web/src/app/shared/components/free-text-filter/free-text-filter.component.scss:9]: I tried, it does not give the same response

File: our-brothers-web/src/app/shared/components/view-toggle/view-toggle.component.html (hunk 6 above)
Comment (KobiHay) [reply our-brothers-web/src/app/shared/components/view-toggle/view-toggle.component.html:1]: fix

File: our-brothers-web/src/app/shell/header/header.component.scss (hunk 7 above)
Comment (KobiHay) [reply our-brothers-web/src/app/shell/header/header.component.scss:15]: no, If you change the parameter the interval will also change which is not good, we only need to change the height of the line

**Pull request merged**